test/concurrent/emit-interrupt-stop.cc

```diff
@@ -18,29 +18,25 @@ TYPED_TEST(ConcurrentTypedTest, EmitInterruptStop) {
     return Stream<int>()
                .interruptible()
                .begin([](auto& k, Interrupt::Handler& handler) {
-                 handler.Install([&k]() {
-                   k.Stop();
-                 });
+                 handler.Install([&k]() { k.Stop(); });
```

Comment:
Please don't introduce formatting changes unless they are to make the codebase more consistent.